README.md

```diff
@@ -1,24 +1,170 @@
-# README
+<a name="readme-top"></a>
 
```

Comment:
- [ ] In your `.gitignore` file you ignored `config/database.yml` file used for setting up database. Kindly restore this file. Ignoring this file makes setting up your project tedious. Across teams, that is not good practice. If you have variables you do not wish to expose then you should use the environment variables for them.